wled00/colors.cpp

```diff
@@ -38,13 +46,9 @@ uint32_t color_add(uint32_t c1, uint32_t c2, bool preserveCR)
   uint32_t g = wg & 0xFFFF;
 
   if (preserveCR) { // preserve color ratios
-    uint32_t max = std::max(r,g); // check for overflow note
+    uint32_t max = std::max(r,g); // check for overflow
```

Comment:
well this line doesn't check anything. I'd move the comment up one line (or down to the if)